library/BIGBITMAPS

```diff
@@ -65,11 +69,11 @@
 
 (PUTPROPS |\\SFInvert| MACRO ((|BitMap| \y)
 
-         (* |corrects| |for| |the| |fact| |that| |alto| |bitmaps| |are| |stored| |with| 
```

Comment:
can we convert these comments to strings? Or use a different readtable than XCL?